devops/infrastructure/main.yml

```diff
@@ -161,6 +161,44 @@ Resources:
       Timeout: '600'
       Count: 4
 
+  S3Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: PublicRead
+      WebsiteConfiguration:
+        IndexDocument: index.html
+
+  BucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      PolicyDocument:
+        Id: PublicPolicy
+        Version: 2012-10-17
+        Statement:
+          - Sid: PublicReadForGetBucketObjects
+            Effect: Allow
+            Principal: '*'
+            Action: 's3:GetObject'
+            Resource: !Sub "arn:aws:s3:::${S3Bucket}/*"
+      Bucket: !Ref S3Bucket
+
+  CloudFrontDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Origins:
+        - DomainName: !Select [1, !Split ["//", !GetAtt S3Bucket.WebsiteURL]]
+          Id: pioneer-origin-s3
+          CustomOriginConfig:
+            OriginProtocolPolicy: http-only
```

Comment:
https only would be better, since most browsers now have a default https only?

Reply:
`OriginProtocolPolicy` specifies the protocol (HTTP or HTTPS) that CloudFront uses to connect to the origin (S3 bucket). Since, S3 only supports HTTP for websites and since this is an internal communication, we use HTTP.

The client side will use HTTPS (set by `ViewerProtocolPolicy`).